src/frontends/paddle/src/op/dequantize_linear.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs dequantize_linear(const NodeContext& node) {
+    // extract the INPUTS
+    const auto x = node.get_input("X");
+    const auto scale = node.get_input("Scale");  // type: float or 1-D
+    const auto zero_point = node.get_input("ZeroPoint");
+
+    // assert shape of scale and zero_point
+    PADDLE_OP_CHECK(node, scale.get_partial_shape().rank().is_static(), "dequantize: scale rank must be static!");
+    PADDLE_OP_CHECK(node,
+                    zero_point.get_partial_shape().rank().is_static(),
+                    "dequantize: zero_point rank must be static!");
+    const auto& scale_shape = scale.get_partial_shape();
+    const auto& scale_shape_length = scale_shape.rank().get_length();
+
+    if (scale_shape_length == 1) {
+        PADDLE_OP_CHECK(node,
+                        scale.get_partial_shape() == zero_point.get_partial_shape(),
```

Comment:
Would it be better to use compatible utility?

Reply:
> Would it be better to use compatible utility?

Maybe not. The reason is the shape check is very simple. The partial_shape::compatible is a bit heavy.